projects/VitalRecord/CodeSystems.cs

```diff
@@ -64,15 +62,25 @@ public static class CodeSystems
         /// <summary> HL7 Yes No </summary>
         public static string YesNo = "http://terminology.hl7.org/CodeSystem/v2-0136";
 
+        /// <summary> USPS </summary>
+        public static string USPS = "https://www.usps.com";
+
+        /// <summary> NAHDO </summary>
+        public static string NAHDO = "https://nahdo.org/sopt";
+
+        /// <summary> ActReason </summary>
+        public static string ActReason = "http://terminology.hl7.org/CodeSystem/v3-ActReason";
+
+        // Code systems defined within vital record IGs
+
         /// <summary> Bypass Edit Flag </summary>
         public static string BypassEditFlag = "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs";
 
         /// <summary> Pregnancy Status </summary>
         public static string PregnancyStatus = "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-pregnancy-status-cs";
 
         /// <summary> Missing Value Reason </summary>
-        public static string MissingValueReason = "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-missing-value-reason-cs";
-
+        public static string MissingValueReason = "http://hl7.org/fhir/us/vr-common-library/CodeSystem/CodeSystem-missing-value-reason-vr";
```

Comment:
These are some of the CodeSystems that would be of concern in VRDR

Reply:
Fixed. Where there are two code systems with similar names but a new place I name the new ones with a VRCL prefix.